userguide/tutorials/aws-singletier.adoc

```diff
@@ -227,6 +227,7 @@ The `diagnostic` option of the `kpm` command creates an extensive report for a g
   --killbill-api-credentials=KEY SECRET \
   --kaui-web-path=/var/lib/tomcat/webapps2 \
   --killbill-url=http://127.0.0.1:8080 \
+  --account-export=ACCOUNT_ID \
   --database-host=127.0.0.1:3306
```

Comment:
Same comment as above, Jim would be incorporating this change as part of his work.

Reply:
Hi @reshmabidikar this one too.